src/library_fs.js

```diff
@@ -669,6 +665,15 @@ FS.staticInit();` +
       mode = mode !== undefined ? mode : 511 /* 0777 */;
       mode &= {{{ cDefine('S_IRWXUGO') }}} | {{{ cDefine('S_ISVTX') }}};
       mode |= {{{ cDefine('S_IFDIR') }}};
+#if FS_DEBUG
+      try {
+        if (FS.trackingDelegate['onMakeDirectory']) {
+          FS.trackingDelegate['onMakeDirectory'](path, mode);
+        }
+      } catch(e) {
+        err("FS.trackingDelegate['onMakeDirectory']('" + path + "', " + mode + ") threw an exception: " + e.message);
```

Comment:
Do you know why we both to try/catch and ignore and log exceptions in these callbacks?  It seems to make the tacking code more complex and I'm not sure why its a good idea.

Reply:
I think originally this was done for debugging callbacks. We can just leave it up to the user to handle errors in the functions that they defined. Will remove the try/catch.